metaci/release/utils.py

```diff
@@ -13,12 +13,17 @@
 logger = logging.getLogger(__name__)
 
 
-def implementation_payload(role, config_item):
-    if role and config_item:
+def implementation_payload(role, config_item, release):
+    if role and config_item and release:
         return {
             "description": role,
-            "start_time": role,
-            "end_time": role,
+            "owner": "00XXXXXXXXXX",  # need to tie GUS user id to MetaCI user.
```

Comment:
I think we need to sort this out before we can merge this. Otherwise trying to create the case will fail, no?

Reply:
@davisagli what do you propose passing it through a variable?

Reply:
@jofsky Do you think we can just use our GUS API user for all of them?

Reply:
I was hoping to map it to each ERE. Apply PLOP and only give those users in metaCI mappings if possible.

Reply:
Sure, that would be ideal but is more work so I'm wondering if we can use the API user for now, and then improve it later, rather than adding to the scope of this PR.